src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public static final int MAGIC_NUMBER = 47;
```

Comment:
It seems that the constant 'MAGIC_NUMBER' is a magic number used to seed the random number generator. This name does not convey the purpose of the constant. A more descriptive name would be beneficial. For example, 'RANDOM_SEED' would indicate that this number is used as a seed for random number generation.